index.html

```diff
@@ -215,10 +215,10 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
           
               <div class="grid-item">
                 <div class="forms">
-                    <form action="#" method="GET">
-                        <input type="text" placeholder="Full name">
-                        <input type="email" placeholder="Email adress">
-                        <textarea placeholder="Enter text here"></textarea>
+                    <form action="#" method="POST">
+                        <input type="text" placeholder="Full name" maxlength="30" required>
```

Comment:
- [ ] Your form is not working. Please use [Formspree](https://formspree.io/html/) in your form. :+1: You see the instructions from this [link](https://formspree.io/html/)